main.go

```diff
@@ -110,6 +114,12 @@ func main() {
 		"The max allowed size in bytes of a file in a Helm chart.")
 	flag.DurationVar(&requeueDependency, "requeue-dependency", 30*time.Second,
 		"The interval at which failing dependencies are reevaluated.")
+	flag.IntVar(&cacheMaxSize, "cache-max-size", 0,
+		"The maximum size of the cache in number of items.")
+	flag.StringVar(&cacheTTL, "cache-ttl", "15m",
+		"The TTL of an item in the cache. Valid time units are ns, us (or µs), ms, s, m, h.")
+	flag.StringVar(&cachePurgeInterval, "cache-purge-interval", "1m",
+		"The interval at which the cache is purged. Valid time units are ns, us (or µs), ms, s, m, h.")
```

Comment:
I would make these flags Helm specific, and prefix them either with `helm-` or `helm-index-`. In addition, I think they should be mentioned somewhere in the spec, maybe in a "Working with" section related to Helm chart objects about controller resource consumption.